src/config/ImageUrls.ts

```diff
@@ -0,0 +1,5 @@
+const cdnUrl = 'https://cdn.sharechat.com';
+export const ImageUrl = {
+    CALENDAR: `${cdnUrl}/react-native-assets/moj/cashout/calendar.png`,
+    CLOCK_ICON: `${cdnUrl}/react-native-assets/moj/recharge-list/clock.png `,
+};
```

Comment:
This is not correct, this is a library, these icons should be access as prop from outside, we can't use sharechat cdn url